ci/std/go/dagger.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "go",
+  "sdk": "go",
+  "dependencies": [
+    {
+      "name": "daggy",
+      "source": "github.com/shykes/daggerverse/daggy@dfb1f91fa463b779021d65011f0060f7decda0ba"
+    },
+    {
+      "name": "wolfi",
+      "source": "github.com/shykes/daggerverse/wolfi@dfb1f91fa463b779021d65011f0060f7decda0ba"
+    }
+  ],
```

Comment:
I guess this is more of a philosophical question - but should our standard library here consume other modules in the daggerverse?

My personal opinion is we should avoid this, and try and be a "root" that others could pull from - but maybe you disagree.

Reply:
I think that's a good goal. I was planning on pulling in these modules as a follow-up PR.

Also, I don't know why we're importing `daggy`.

Reply:
> Also, I don't know why we're importing daggy.

Removed